publications-2011.bib

```diff
@@ -121,7 +121,8 @@ @article{2011_10
 @article{2011_11,
    author  = {S. Chandrasekaran and K. R. Jayaraman and M. Gu and H. N. Mhaskar and J. Moffitt},
    title   = {MSNFD: A Higher Order Finite Difference Method for Solving Elliptic PDEs on scattered points (An Extended version of our paper submitted to ICCS 2011)},
-   journal = {submitted},
+   journal = {},
+   note = {Submitted},
```

Comment:
this means the word "submitted" will not show up in the html anymore, right?

Reply:
Hmm... I guess not. It does in the PDF but not the HTML (at least not with our custom output; with Jabref's default export option it did as well).

Reply:
I think we should leave "submitted" inside the journal and just add the "note"

Reply:
That's what I had tried to do in #25: to put "submitted" along with the journal. 

But we could add the "note" field to the output. jabref doesn't care whether fields are standardized. If you do 
```
  \begin{note} \format[HTML]{\note} \end{note}
```
it will happily copy the note somewhere.

Reply:
@bangerth can we add this somewhere at the end (maybe even in a separate line)

Reply:
You mean add the note to the HTML output? Yes, it's all possible. I can do that later this afternoon, but let's leave that for a different PR.

Reply:
> You mean add the note to the HTML output?

yes.

> let's leave that for a different PR.

of course!

Reply:
> You mean add the note to the HTML output?

I like the idea too!